src/RadhydroSimulation.hpp

```diff
@@ -509,21 +509,20 @@ template <typename problem_t> void RadhydroSimulation<problem_t>::addStrangSplit
 template <typename problem_t>
 auto RadhydroSimulation<problem_t>::addStrangSplitSourcesWithBuiltin(amrex::MultiFab &state, int lev, amrex::Real time, amrex::Real dt) -> bool
 {
-
-	// start by assuming chemistry burn is successful.
-	bool burn_success = true; // NOLINT
-
+	bool cool_success = true;
```

Comment:
It would be good to add a comment before this line that says something similar to 'start by assuming chemistry burn is successful'